scassandra/src/main/scala/com/evolutiongaming/scassandra/ResultSet.scala

```diff
@@ -0,0 +1,80 @@
+package com.evolutiongaming.scassandra
+
+import cats.effect.{Concurrent, Sync}
+import cats.effect.implicits._
+import cats.syntax.all._
+import com.datastax.driver.core.{Row, ResultSet => ResultSetJ}
+import com.evolutiongaming.scassandra.util.FromGFuture
+import com.evolutiongaming.sstream.Stream
+import com.evolutiongaming.sstream.FoldWhile._
+
+
+object ResultSet {
```

Comment:
in order to not conflict with `ResultSet` class of cassandra, could we rather add this is implicit extension method to `ResultSet` in `syntax.scala` ?

Reply:
Great idea, let's transfer this extension to `scassandra.syntax`.

Reply:
@t3hnar Done.